assets/global.js

```diff
@@ -8,7 +8,7 @@ function getFocusableElements(container) {
 
 document.querySelectorAll('[id^="Details-"] summary').forEach((summary) => {
   summary.setAttribute('role', 'button');
-  summary.setAttribute('aria-expanded', 'false');
+  summary.setAttribute('aria-expanded', summary.parentNode.hasAttribute('open'));
```

Comment:
This is a good point about other parts of the theme 🤔 the reason I added this is because by default the first summary is open. So if we do this, we are always setting it to `false` and in this case the first one is open.

https://screenshot.click/23-39-ov5ns-a67ed.png
Otherwise by default this would be false: https://screenshot.click/23-40-jcfhx-u44zd.png